src/sentry/static/sentry/app/components/charts/eventsChart.tsx

```diff
@@ -88,7 +88,10 @@ class Chart extends React.Component<ChartProps, State> {
     return true;
   }
 
-  getChartComponent() {
+  getChartComponent():
+    | React.ComponentType<BarChart['props']>
```

Comment:
I was running into `JSX element type 'Component' does not have any construct or call signatures` with dynamic chart components now that they are all converted. Someone lmk if there is a better option.